recipe/meta.yaml

```diff
@@ -19,96 +19,52 @@ source:
+    - gnuconfig                           # [unix]
+    - make                                # [unix]
+    - pkg-config                          # [not win]
     - sip 6.7.11
+    - setuptools
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ cdt('gtk2-devel') }}            # [linux]
-    - {{ cdt('libpng-devel') }}          # [linux]
-    - {{ cdt('libxrender-devel') }}      # [linux]
-    - {{ cdt('mesa-libgl-devel') }}      # [linux]
-    - {{ cdt('mesa-dri-drivers') }}      # [linux]
-    - {{ cdt('libXxf86vm-devel') }}      # [linux]
-    - {{ cdt('libselinux') }}            # [linux]
-    - {{ cdt('libdrm-devel') }}          # [linux]
-    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
-    - {{ cdt('libxau-devel') }}          # [linux]
-    - {{ cdt('libx11-devel') }}          # [linux]
-    - {{ cdt('libxfixes-devel') }}       # [linux]
-    - {{ cdt('libxext-devel') }}         # [linux]
-    - {{ cdt('libxinerama') }}           # [linux]
-    - {{ cdt('libxi') }}                 # [linux]
-    - {{ cdt('libxcb') }}                # [linux]
-    - {{ cdt('libxcursor') }}            # [linux]
-    - {{ cdt('libxcomposite') }}         # [linux]
-    - {{ cdt('libxdamage-devel') }}      # [linux]
-    - {{ cdt('webkitgtk-devel') }}       # [linux]
-    - {{ cdt('libsoup-devel') }}         # [linux]
+    - {{ stdlib("c") }}
+    - {{ cdt('mesa-libgl-devel') }}       # [linux]
+    - {{ cdt('mesa-dri-drivers') }}       # [linux]
+    - {{ cdt('libselinux') }}             # [linux]
+
   host:
     - python
     - pip
     - attrdict3
-    - cairo  # [linux]
     - cython >=0.29.32,<3
-    - expat  # [not win]
-    - gettext  # [not win]
-    - glib  # [linux]
-    - gst-plugins-base {{ gstreamer }}  # [linux]
-    - gstreamer                         # [linux]
-    - gtk2  # [linux]
-    - libjpeg-turbo   # [not win]
-    - libglu  # [linux]
-    - libiconv  # [osx]
-    - libpng  # [not win]
-    - libtiff  # [not win]
-    - libxml2  # [linux]
-    - pango  # [linux]
-    - pathlib2
-    - pcre2  # [not win]
-    - pthread-stubs
+    - gtk3                # [linux]
     - requests
     - setuptools
-    - xz  # [not win]
-    - zlib  # [not win]
+    - wxwidgets
+    - xorg-xineramaproto  # [linux]
+    - xorg-libxfixes      # [linux]
+    - xorg-libxext        # [linux]
+    - xorg-libxxf86vm     # [linux]
+    - xorg-libxdamage     # [linux]
 
   run:
     - python
-    - cairo  # [linux]
-    - expat  # [not win]
-    - gettext  # [not win]
-    - glib  # [linux]
-    - gtk2  # [linux]
-    - libjpeg-turbo   # [not win]
-    - libglu  # [linux]
-    - libiconv  # [osx]
-    - libpng  # [not win]
-    - libtiff  # [not win]
-    - libxml2  # [linux]
+    - python.app       # [osx]
+    - gtk3             # [linux]
     - numpy
-    - pango  # [linux]
-    - pathlib2
-    - pcre2  # [not win]
     - pillow
-    - pthread-stubs
-    - pypubsub  # [py3k]
-    - python.app  # [osx]
-    - requests
```

Comment:
some of these are python libraries, why is it ok to remove them now?

Reply:
Just bringing the recipe up to date with respect to the upstream requirements: https://github.com/wxWidgets/Phoenix/blob/master/requirements/install.txt
 - wxPython vendors pypubsub: https://github.com/wxWidgets/Phoenix/blob/master/wx/lib/pubsub/README_WxPython.txt
 - `requests` is only used at build time (it's only referenced from `build.py`)
 - `pathlib2` is only used at build time, and only as a fall back if the stdlb `pathlib` is not present (i.e. for old versions of Python)

Reply:
Great. Thank you for confirming